web_src/js/utils/match.ts

```diff
@@ -41,3 +44,15 @@ export function matchMention(queryText: string): string[] {
 
   return sortAndReduce(results);
 }
+
+export async function matchIssue(owner: string, repo: string, _issueIndex: string, queryText: string): Promise<Issue[]> {
+  const query = queryText.toLowerCase();
```

Comment:
I believe it is the API's responsibility to handle the case casting .....

Reply:
moved it to the api